reference/latest/build.gradle

```diff
@@ -1,6 +1,6 @@
 def minecraftVersion = "1.21"
 def yarnVersion = "1.21+build.7"
-def fabricApiVersion = "0.100.4+1.21"
```

Comment:
cc. @modmuss50

Reply:
Whats the issue? This seems fine?

Reply:
Could maybe bump this to 1.21.1, and the latest FAPI but it doesnt make a lot of difference.